source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,209 @@
+
+.. _security:
+
+|CL| Security 
+*********************
+
+|CL|, as it is distributed and maintained upstream, aims to make 
+systemic and layered security conscious decisions, which are both
+performant and practical. This security philosophy is rooted within
+the project's codebase and operating culture.
+
+Below are just some examples of how:
+
+
+.. contents:: :local:
+   :depth: 2
+
+
+
+Security in Updates
+===================
+
+The |CL| team believes in the potential benefits of 
+software security through open sourcing, incremental updates, and 
+rapidly resolving known security advisories.
+ 
+
+
+The latest Linux codebase
+--------------------------------
+
+The |CL| Project operates around the idea of open sourcing 
+ideas and innovations upstream through the Linux kernel and 
+using it to take advantage of the latest and greatest features. 
+This includes security-centered innovations adopted in the Linux upstream. 
+
+ 
+
+Automated Effective Updating
+----------------------------
+
+The |CL| upstream is updated at lightning pace, releasing full
+new versions of |CL| with incremental updates multiple times a day! 
+
+This `rolling release model`_ allows |CL| to consume the latest security
+fixes of software packages as soon as they become available. 
+There is no waiting for major or minor releases on |CL|. 
+
+But an update is not effective if it is just simply downloaded onto a system. 
+It needs to be obtained *AND* ensured that the new patched copy is being
+used; not an older copy loaded into memory. |CL| will let you know when a 
+service needs to be rebooted or do it for your automatically after 
+a software update, if desired.
+
+
+In |CL| updates are delivered automatically, efficiently, 
+and effectively.For more information see `documentation about Software Updates`_ in |CL|.
+
+ 
+
+
+
+Automated CVE Scanning and Remediation
+--------------------------------------
+
+The sheer number of software packages and security vulnerabilities is growing 
+at an exorbitant rate. Repositories of CVEs and their fixes, if known, 
+are published by :abbr:`NIST` in a National Vulnerability Database 
+\ |NVD|\  and at \ |MITRE|\  .
+
+
+|CL| takes a proactive and measured approach to addressing known 
+and fixable Common Vulnerabilities and Exposures (CVEs) disclosures.
+|CL| packages are scanned against CVEs at least daily, and 
+software patched in accordance to the fix. This scanning is automated 
```

Comment:
software patched in accordance to the fix. -> security updates and patches are deployed as soon as they are available.

Reply:
Remove the last sentence on this paragraph (This scanning...) as it will be out-of-date very shortly.